di-ipv-credential-issuer-stub/deploy/bav/template.yaml

```diff
@@ -605,6 +638,19 @@ Resources:
                   - !Sub "arn:aws:ssm:${AWS::Region}:${AWS::AccountId}:parameter/stubs/credential-issuer/*"
                   - !Sub "arn:aws:ssm:${AWS::Region}:${AWS::AccountId}:parameter/stubs/credential-issuer-stub-clients"
                   - !Sub "arn:aws:ssm:${AWS::Region}:${AWS::AccountId}:parameter/stubs/credential-issuer-stub-clients/*"
+        - PolicyName: GetDynatraceSecret #pragma: allowlist secret
+          PolicyDocument:
+            Version: 2012-10-17
+            Statement:
+              - Effect: Allow
+                Action: secretsmanager:GetSecretValue #pragma: allowlist secret
+                Resource: arn:aws:secretsmanager:eu-west-2:216552277552:secret:* #pragma: allowlist secret
+              - Effect: Allow #pragma: allowlist secret
+                Action: secretsmanager:ListSecrets #pragma: allowlist secret
```

Comment:
I'm a bit surprised that we'd need this permission - doesn't the code know the names of the secrets it's going to access?

Reply:
Also around line 589